kernel-module/build-for-jetson-drive-igpu-native.sh

```diff
@@ -20,4 +20,22 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+KERNEL_SRC_DIR="/usr/src/nvidia/nvidia-oot/"
+if [ ! -d ${KERNEL_SRC_DIR} ]; then
+	echo "Kernel source not found in $KERNEL_SRC_DIR"
+	KERNEL_SRC_DIR="/usr/src/./linux-headers-tegra-oot*"
+	if [ ! -d ${KERNEL_SRC_DIR} ]; then
+		echo "Kernel source not found in $KERNEL_SRC_DIR"
+		exit;
+	fi
+fi
+echo "Kernel source ${KERNEL_SRC_DIR}"
```

Comment:
THis is nvidi oot source so has to be echoed the same.

Reply:
same updated message Nvidia OOT  source